source/_integrations/zwave_js.markdown

```diff
@@ -649,83 +679,73 @@ Node {{ node }};{{ s.name }};{{ s.entity_id }}{% endfor %}
 
-<div class='note info'>
-While adding devices, you have the option to use `secure inclusion`, this means that the traffic between the controller and the device will be encrypted. This adds additional overhead to the Z-Wave network so use this option with care. As a general rule of thumb it is advised to only securely include devices that actually NEED this kind of protection. A good example is a Z-Wave door lock.
-</div>
+If your device is included using S2 security, you may be prompted to enter a PIN number provided with your device. Often, this PIN is provided with the documentation _and_ is also printed on the device itself.
+
+**Warning:**
+
+1. **Do not move your Z-Wave stick to include devices.** This is no longer necessary and leads to broken routes. 
+2. **Do not initiate device inclusion from the Z-Wave stick itself.** This is no longer supported.
+
+#### Should I use `Secure Inclusion`?
 
-### How can I remove (exclude) a device from my Z-Wave network?
+That depends. There are two generations of Z-Wave security, S0 and S2. 
+
+S0 security imposes significant additional traffic on your mesh and is recommended only for devices that require security, such as door locks.
+
+S2 security does not impose additional network traffic and provides additional benefits, such as detecting packet corruption. By default, Z-Wave JS attempts S2 security during inclusion if supported, falling back to S0 security only when necessary.
+
+#### How do I remove (exclude) a device from my Z-Wave network?
 
 1. In Home Assistant: open Configuration -> Integrations -> Z-Wave JS -> Configure.
 2. Press `Remove node`.
 3. Press `Start Exclusion`. The Z-Wave controller is now in exclusion mode and will not respond to other commands.
 4. Put the device you want to remove in exclusion mode. Refer to its manual how this is done.
 5. The UI should confirm that the node was removed and the device and entities will be removed from Home Assistant.
 
-### Where do I need to enter the network key?
+### Troubleshooting
 
-- You enter the network key during integration setup if you do not have the add-on installed.
-- Official Z-Wave JS add-on: In the add-on configuration, directly in the supervisor.
-- Z-Wave JS 2 MQTT: In the web UI, go to Settings -> Z-Wave -> Network Key.
+#### I'm having a problem, what should I do first?
 
-### How can I use my OZW network key in zwavejs2mqtt?
+_Many_ reported issues result from RF interference caused by the system's USB ports. This can manifest in many ways, including devices that won't include at all, devices that won't include securely, sensors with erroneous values (packets corrupted), delayed controlled of devices, or no ability to control devices. These problems can be intermittent, and they may be newly apparent after switching from `ozw`, though they existed before, as Z-Wave JS interviews devices and reports errors differently.
```

Comment:
```suggestion
_Many_ reported issues result from RF interference caused by the system's USB ports. This can manifest in many ways, including devices that won't include at all, devices that won't include securely, sensors with erroneous values (packets corrupted), delayed control of devices, or no ability to control devices. These problems can be intermittent, and they may be newly apparent after switching from `ozw`, though they existed before, as Z-Wave JS interviews devices and reports errors differently.
```